sympy/physics/tests/test_physics_matrices.py

```diff
@@ -70,8 +69,10 @@ def test_Dirac():
 
 def test_mdft():
     assert mdft(1) == Matrix([[1]])
-    assert mdft(2) == 1/sqrt(2)*Matrix([[1,1],[1,-1]])
+    assert mdft(2) == 1/sqrt(2)*Matrix([[1,1],[1,exp(-I*pi)]])
```

Comment:
Isn't `exp(-I*pi)` equal to `-1`? Why is this change needed?

Reply:
@certik If I'm not mistaken, such expressions are not simplified in `SymEngine`.

Reply:
I see. I think that's an omission, I just created an issue for it: https://github.com/symengine/symengine/issues/1330